src/XIOPAK.CPP

```diff
@@ -34,7 +34,9 @@
 /*------------------------------- INCLUDES --------------------------------*/
 
 #include <io.h>		// open, read, write,
+#if !defined( ELIMINATE_WINDOWS)
 #include <direct.h>	// getcwd, chdir, _chdrive
+#endif
```

Comment:
Can we remove this entirely? With the move to file_system, I don't think anything in this file uses direct.h anymore.

Reply:
Yes, we can. I'll remove it.